sagan-site/src/main/java/sagan/projects/support/ProjectMetadataController.java

```diff
@@ -33,4 +40,87 @@ public Project projectMetadata(@PathVariable("projectId") String projectId) thro
         return service.getProject(projectId);
     }
 
+    @RequestMapping(value = "/{projectId}/**", method = GET)
+    public ProjectRelease releaseMetadata(@PathVariable("projectId") String projectId) throws IOException {
+        Project project = service.getProject(projectId);
+        if (project == null) {
+            throw new MetadataNotFoundException("Cannot find project " + projectId);
+        }
+        List<ProjectRelease> releases = project.getProjectReleases();
+        String path = ServletUriComponentsBuilder.fromCurrentRequest().build().getPath();
+        String version = path.substring(("/project_metadata/" + projectId + "/").length());
+        for (ProjectRelease release : releases) {
+            if (release.getVersion().equals(version)) {
+                return release;
+            }
+        }
+        throw new MetadataNotFoundException("Could not find " + version + " for " + projectId);
+    }
+
+    @RequestMapping(value = "/{projectId}", method = POST)
```

Comment:
You're reading it wrong. It's appending or updating a release to an existing project. I think POST is better.